nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkCommandJsonReader.java

```diff
@@ -54,13 +52,13 @@ public BulkCommandJsonReader() {
     }
 
     @Override
-    protected BulkCommand readEntity(JsonNode jn) {
+    protected BulkCommand readEntity(JsonNode jn) throws IOException {
         // everything is mandatory except parameters
         Function<String, String> getter = fieldName -> jn.get(fieldName).asText();
 
         Map<String, Serializable> params = emptyMap();
         if (jn.has(COMMAND_PARAMS)) {
-            params = toMap(jn.get(COMMAND_PARAMS));
+            params = BulkParameters.paramsToMap(new ObjectMapper().writeValueAsString(jn.get(COMMAND_PARAMS)));
```

Comment:
Can you create a `BulkParameters#paramsToMap(JsonNode)` which will allow you to not serialise then deserialise the JsonNode.

Reply:
Not resolved.